frontend/taipy/src/DataNodeViewer.tsx

```diff
@@ -756,6 +757,29 @@ const DataNodeViewer = (props: DataNodeViewerProps) => {
                                 ) : null}
                             </Stack>
                         </Box>
+                        <div>
+                            { dtType === "json" ? (
```

Comment:
Did you check that this type is set by the backend ?

Reply:
No i didn't checked as on the issue the user was using json data so i thought i will make it in a condition so that only json data will be collapsed.

Reply:
Here is what is a [taipy.core.data_node ](https://docs.taipy.io/en/latest/userman/scenario_features/data-integration/data-node-usage/)

A data_node can contain any type of data (scalar, tabular, structured, db view ...)
the data_node component tries to display a data_node and its underlying data.

Reply:
as per https://github.com/Avaiga/taipy/blob/63f0cbf9a5e6346e54c5c4c0071ee8dc2cca8828/taipy/gui_core/_adapters.py#L205

the best we can hope for is a `dict` value for `dtType` and that's what we get if the `DataNode` used is a `JSONDataNode`.
Described here
https://github.com/Avaiga/taipy/blob/9373005365e32d9bbc70d05f455c32475c7cf44e/taipy/core/data/json.py#L141

Reply:
Thanks for sharing this i was having a really hard time with the backend.